lib/stella/helpers.rb

```diff
@@ -0,0 +1,103 @@
+require 'typhoeus/adapters/faraday'
+
+module Stella
+  module Helpers
+    extend ActiveSupport::Concern
+
+    @@types = []
+
+    included do
+      if defined? Rails
+        env = Rails.env
+      else
+        env = 'test'
+      end
+
+      index_name [search_index_name, env].join('_')
```

Comment:
I'll refactor this to add a hook which will allow for override of the naming scheme.